test/controllers/games_controller_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class GamesControllerTest < ActionController::TestCase
+  test "should get show" do
+    get :show
+    assert_response :success
```

Comment:
It's needs a game id to pass. Should I go ahead and add FactoryGirl and disable fixtures?